# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLContext.h
# Commit: 05896dbeb434
# Full Hash: 05896dbeb434d6f3ca9324f01eb67020d932f50f
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-15 21:48:38
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - Determine webgl's layers::TextureType in Present. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D79617
# ==============================================================================

diff -r 8439f89e170b -r 05896dbeb434 dom/canvas/WebGLContext.h
--- a/dom/canvas/WebGLContext.h	Mon Jun 15 18:26:12 2020 +0000
+++ b/dom/canvas/WebGLContext.h	Mon Jun 15 18:26:14 2020 +0000
@@ -488,8 +488,7 @@
  public:
   void Present(WebGLFramebuffer*, layers::TextureType);
   RefPtr<gfx::DataSourceSurface> GetFrontBufferSnapshot();
-  Maybe<layers::SurfaceDescriptor> GetFrontBuffer(WebGLFramebuffer*,
-                                                  layers::TextureType);
+  Maybe<layers::SurfaceDescriptor> GetFrontBuffer(WebGLFramebuffer*);
 
   void RunContextLossTimer();
   void CheckForContextLoss();